boms/user/server/ee/preview/pom.xml

```diff
@@ -70,6 +83,20 @@
                             <bomVersion>${project.version}</bomVersion>
                             <bomName>${full.dist.product.release.name} Preview BOMs: EE</bomName>
                             <bomDescription>${full.dist.product.release.name} Preview: EE Dependency Management</bomDescription>
+                            <channels>
+                                <channel>
+                                    <manifest>
+                                        <groupId>org.wildfly.channels</groupId>
+                                        <artifactId>wildfly-ee</artifactId>
+                                    </manifest>
+                                </channel>
```

Comment:
This shouldn't be used. Standard WF and WFP are separate.

Reply:
fixed